### PR TITLE
remove files/tmp/.metadata_never_index

### PR DESCRIPTION
**Pull request by kellyselden**
Is this an accidental check in?

### CODE REVIEW

**Comment by rwjblue** [comment]
Nope 😸

**Comment by rwjblue** [comment]
> these files are used to prevent indexing programs from automatically indexing the metadata. Programs, such as Apple Spotlight incorporate advanced searching options on Macintosh computers for faster searching, indexing and launching of applications. But, what if the user does not want a certain hard disk partition, directory or folder to be indexed? This can be achieved by using a .metadata_never_index file, which instructs Apple Spotlight to skip a certain partition, directory or folder.

**Comment by kellyselden** [comment]
maxresdefault

**Comment by Turbo87** [comment]
@rwjblue @kellyselden I very much disagree with checking this file into version control! IMHO this should be generated automatically by Ember CLI instead.

**Comment by rwjblue** [comment]
tldr; I didn't think it all the way through, sorry y'all!
